app/src/main/java/io/github/jbellis/brokk/Service.java

```diff
@@ -677,7 +677,7 @@ private boolean supportsReasoning(String location) {
      * @param modelName      The display name of the model (e.g., "gemini-2.5-pro-exp-03-25").
      */
     @Nullable
-    public StreamingChatModel getModel(String modelName, ReasoningLevel reasoningLevel, @Nullable Double temperature) {
+    public StreamingChatModel getModel(String modelName, ReasoningLevel reasoningLevel, @Nullable OpenAiChatRequestParameters.Builder parametersOverride) {
```

Comment:
Decided to go with [option 3](https://github.com/BrokkAi/brokk/issues/531#issue-3257893303):

> pass a Builder to getModel with the parameters you want